packages/model/src/base/tests/dependencies.test.ts

```diff
@@ -0,0 +1,22 @@
+import {PackageJson} from '../dependencies';
+
+describe('PackageJson', () => {
+  it('loads the package.json for its own package @sewing-kit/model', () => {
```

Comment:
This test feels a bit fragile to me. Eg, if `@sewing-kit/model` every removes the `glob` dependency, this test would fail. I would just leave this test out.

Reply:
Yes, I should create a test package.json specifically for the test instead of using a real one.  I wanted to test the two ways of creating a PackageJson object.